master/static/migrations/20231215103902_server-maintain-message.tx.up.sql

```diff
@@ -0,0 +1,7 @@
+CREATE TABLE maintenance_messages (
+  id SERIAL PRIMARY KEY,
```

Comment:
```suggestion
  id integer PRIMARY KEY GENERATED BY DEFAULT AS IDENTITY,
```

`SERIAL` should not be used according to [the Postgres docs](https://wiki.postgresql.org/wiki/Don%27t_Do_This#Don.27t_use_serial).